earthpy/tests/conftest.py

```diff
@@ -1,8 +1,37 @@
 """ Utility functions for tests. """
 import numpy as np
 import pytest
+from shapely.geometry import Polygon
 from affine import Affine
 import rasterio as rio
+import geopandas as gpd
+
+@pytest.fixture
```

Comment:
@nkorinek 
i believe that i moved these fixtures out of conftest.py because they are only used to test the clip module. Did you need them  / use them somewhere else??